src/util/options.schema.json

```diff
@@ -223,6 +223,12 @@
           "items": { "type": "string" },
           "default": []
         },
+        "skipcpp": {
+          "title": "pre.skipcpp",
+          "description": "Skip running the C preprocessor.",
+          "type": "boolean",
+          "default": false
```

Comment:
We should enable this in the `svcomp` conf, but not the older SV-COMP confs because they didn't have that yet.